### DISCUSSION TITLE
Suggestion: ability to change the order of workspaces

### DISCUSSION THREAD

**Discussion by JSGRD22** [zen-browser/desktop#1627] [Ideas]
It would be helpful to be able to change the order of workspaces by dragging them in the workspace selection UI.

**Comment by pwongy** [comment]
Doesn't even have to be drag and drop in my opinion - even an option in the dot menu would be terrific!

**Comment by mikedlr** [comment]
Works for me right now. 

1. right click workspace
2. click on up/down arrow button in top right of dialog
3. use six dot buttons to drag and drop order up and down

**Reply by pwongy** [reply]
Yes this feature has been released now so I think this can be closed.

**Comment by j13ag0** [comment]
Yes, it works, but 
- the interaction provides no feedback
- and what happens if I have 20 workspaces ?

**Reply by szamanr** [reply]
this discussion is old, i believe it was created before reordering was a thing? could you upvote and comment on this recent discussion: https://github.com/zen-browser/desktop/discussions/10485? maybe we can get it noticed by the devs and get some traction on this. :)

**Comment by Beryesa** [comment]
The current iteration with the dots gets really hard to organize when it starts to scroll on the edges
Just a :: drag handle in the vertical workspace list (to appear when clicking reorder workspaces) would be much better

https://github.com/user-attachments/assets/49c5f6e2-87d3-4aac-823f-10ad50469cc8

**Reply by szamanr** [reply]
good point, could you add this to this discussion: https://github.com/zen-browser/desktop/discussions/10485? maybe we can get it noticed by the devs and get some traction on this. :)

this discussion seems like an old one, probably created before reordering was a thing, so not sure if it will be noticed.
